indico/migrations/versions/20171211_0811_03dca79b401c_hidden_folders_support.py

```diff
@@ -0,0 +1,30 @@
+"""Hidden folders support
+
+Revision ID: 03dca79b401c
+Revises: 566d5de4e0e5
+Create Date: 2017-12-11 08:11:23.673487
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '03dca79b401c'
+down_revision = '566d5de4e0e5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic  ###
+    op.add_column('folders', sa.Column('is_always_hidden_in_fe', sa.Boolean(), nullable=False, server_default='false'),
+                  schema='attachments')
+    op.alter_column('folders', 'is_always_hidden_in_fe', server_default=None, schema='attachments')
```

Comment:
Question: Is there something to be done to force the mutual exclusive setting of  hide_in_fe and always_visible at the database level?